### DISCUSSION TITLE
FLUX doesn't understand specifics!

### DISCUSSION THREAD

**Discussion by bsdesigner** [lllyasviel/Fooocus#3561] [General]
I already posted this question on the FORGE discussion page, but thought I'd try to post it here as well:

I came across the fact that Flux doesn't understand many things that SDXL models understood quite well. For example, Flux doesn't understand when I ask it to generate a celebrity, be it a famous actor, musician, politician or something else. Of course it tries, but it doesn't look like them at all.

Also Flux doesn't understand the styles of famous artists, photographers, film directors or architects. If I ask SDXL model to generate me an image in the style of: Mike Deodato or Rembrandt, Fred Herzog or Greg Girard, Aki Kaurismäki or Christopher Nolan, Tadao Ando or Zaha Hadid, SDXL model will cope with it and will generate me an image with maximum hit in the style. But Flux model, generates without absolute understanding of the name of the one whose style should be orientated on....

Maybe I have not downloaded and installed something? Maybe I'm not setting it up right? Is this just my problem or is someone else also facing this problem?...?!

**Comment by IPv6** [comment]
It does not understand them specifically to avoid people generating stuff that reminds celebrities. There is nothing wrong with model, etc, it is just trained that way and it will hardly change in future

**Reply by bsdesigner** [reply]
In other words, she's poorly trained. If a model doesn't understand who Akira Kurosawa and Leonardo DiCaprio are, or what Renaissance and Art Nouveau are, then she's a bad model. She's not fit for serious work...

**Reply by IPv6** [reply]
use something else then. model authors just do not want legal issues - which is quite understandable, imho

**Comment by jaimeastorga2000** [comment]
This is on purpose, obviously. Flux is not trained to reproduce real celebrities, copyrighted characters, artist styles, or NSFW, because those are all areas of legal/ethical/PR concern for [Black Forest Labs](https://blackforestlabs.ai/).

The community is pretty good at producing finetuned models that incorporate those things _eventually,_ but that's on a timescale of _months_. Flux came out three weeks ago. Give it a year and it should be as well developed as SDXL (which, for reference, came out in July 2023).

**Reply by bsdesigner** [reply]
No doubt all we have to do is wait. As always )))

**Reply by DavidDragonsage** [reply]
Flux should at least be able to render artistic periods such as Renaissance and Art Nouveau and the styles of public domain artists such as Rembrandt. There are no legal or ethical concerns in doing so. 

I do not have much experience with Flux at this point, but I found that it seemed pretty clueless about what a dragon was for example, instead creating a plasticized flying seahorse. 😄 It was also weak on rendering ethnicity and even human age keywords.

But it is a very immature platform at this point, I expect it will improve with time. It is capable of creating some very good images.
